_global/js/board_front.js

```diff
@@ -415,14 +598,27 @@ function getPresenceNodeHtml(nodeText, first, trackType, addEnergyRing) {
                     nodeClass = 'card';
                 }
             }, splitOptions);            
-
-            var subText = Capitalise(splitOptions[1]);
+			
+/*             var subText = Capitalise(splitOptions[1]);
             if(splitOptions[1] == 'reclaim-one'){
                 subText = "Reclaim One";
             }
-            subText = Capitalise(splitOptions[0])+", "+subText;
-
-            var top = "";
+            subText = Capitalise(splitOptions[0])+", "+subText; */
```

Comment:
Please actually remove code instead of only commenting it out. The old code can always be found back in git history (that's one of the reasons to use version control in the first place).

Reply:
Yup, makes sense. Learning the etiquette as I go. I thought that the old code would help others see the changes - but Git has me covered there too.